From 513372f3404cfe4cc727348f998e423863112d01 Mon Sep 17 00:00:00 2001
From: Johannes Thoma <johannes@johannesthoma.com>
Date: Tue, 4 Jan 2022 17:39:55 +0100
Subject: [PATCH] Patch 8c56c923  drbd: Fix read access after free from
 upstream DRBD

This should fix our BSOD on drbd_send_confirm_stable.

Remove this patch once upstream is integrated.
---
 drbd/drbd_receiver.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drbd/drbd_receiver.c b/drbd/drbd_receiver.c
index a12b2c5..6aaa33b 100644
--- a/drbd/drbd_receiver.c
+++ b/drbd/drbd_receiver.c
@@ -8560,6 +8560,7 @@ static void conn_disconnect(struct drbd_connection *connection)
 	/* ok, no more ee's on the fly, it is safe to reset the epoch_size */
 	atomic_set(&connection->current_epoch->epoch_size, 0);
 	connection->send.seen_any_write_yet = false;
+	connection->current_epoch->oldest_unconfirmed_peer_req = NULL;
 
 	drbd_info(connection, "Connection closed\n");
 
-- 
2.17.0

